diff --git a/lib/config-array/override-tester.js b/lib/config-array/override-tester.js
--- a/lib/config-array/override-tester.js
+++ b/lib/config-array/override-tester.js
@@ -19,10 +19,8 @@
  */
 
 import assert from "node:assert";
 import path from "node:path";
 import util from "node:util";
-import minimatch from "minimatch";
-
-const { Minimatch } = minimatch;
+import { minimatch, Minimatch } from "minimatch";
 
 const minimatchOpts = { dot: true, matchBase: true };
